helm/prometheus-operator-app/values.yaml

```diff
@@ -788,6 +788,34 @@ grafana:
     #   replacement: $1
     #   action: replace
 
+  ## Enable persistence using Persistent Volume Claims
+  ## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  persistence:
+    type: pvc
+    enabled: false
+    # storageClassName: default
```

Comment:
Feels weird that some setting are commented out and other not. I would say we comment out all default, WDYT?

Reply:
It's all a default so I can just comment everything it you rthink it makes sense

Reply:
Ok, let's comment everything then. But I agree it's good to have visibility for us and customer :)

Reply:
Everything is commented out. I'm sad upstream disagreed with this though